subprojects/docs/src/docs/userguide/writing_build_scripts.adoc

```diff
@@ -13,27 +13,27 @@
 // limitations under the License.
 
 [[writing_build_scripts]]
-== Writing Build Scripts
+= Writing Build Scripts
 
 This chapter looks at some of the details of writing a build script.
 
 [[sec:the_gradle_build_language]]
-=== The Gradle build language
+== The Gradle build language
 
 Gradle provides a _domain specific language_, or DSL, for describing builds. This build language is based on Groovy, with some additions to make it easier to describe a build.
 
 A build script can contain any Groovy language element.footnote:[Any language element except for statement labels.] Gradle assumes that each build script is encoded using UTF-8.
```

Comment:
Missing a period before `Gradle assumes that each build script is encoded using UTF-8.`

Reply:
Seems like when you click the generated footnote at the bottom of the page it can't jump to the correct location in webpage.

Reply:
@blindpirate 

> Seems like when you click the generated footnote at the bottom of the page it can't jump to the correct location in webpage.

Can you give more detail on this? When I try this, it jumps to the bottom of the page as expected. Seeing something different?